playwright/_impl/_browser.py

```diff
@@ -154,6 +159,14 @@ async def close(self) -> None:
             if not is_safe_close_error(e):
                 raise e
 
+    def _notify_remote_closed(self) -> None:
+        # Emulate all pages, contexts and the browser closing upon disconnect.
```

Comment:
I.e I would expect this to be called on web socket disconnect somehow.

Reply:
the naming came originally from Java, but open to change.

Reply:
So I looked at the TS version, looks like you don't need any special code here, just close the browser. But our server seems to be missing the connection termination upon close for some reason.